test/inductor/test_max_autotune.py

```diff
@@ -578,6 +578,32 @@ def f(x, weight):
     def test_empty_conv_input_with_1x1_kernel(self):
         self.test_empty_conv_input(kernel_size=1)
 
+    @config.patch(max_autotune_gemm_backends="TRITON")
+    def test_baddmm(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.weight = torch.nn.Parameter(
+                    torch.randn(64, 64, 192, dtype=torch.float16)
+                )
+                self.bias = torch.nn.Parameter(
+                    torch.randn(64, 1, 192, dtype=torch.float16)
+                )
+
+            def forward(self, x):
+                return torch.ops.aten.baddbmm.default(self.bias, x, self.weight)
+
+        x = torch.randn(
+            64, 2048, 64, dtype=torch.float16, requires_grad=False, device="cuda"
```

Comment:
I'd be happier if test had specially constructed inputs such that bmm would e.g. overflow but `beta<1` would again bring it to the correct range - that way it'd be a clear failure if we were again to mess up decomposition, rather than "the accuracy is worse than original op"

Reply:
Surprisingly, even beta, alpha == 1 was sufficient to cause numerics issue in the @SherlockNoMad 's not upcasted decomposed pr:

```
Exception: Caused by sample input at index 0: SampleInput(input=Tensor[size=(5, 5, 10), device="cuda:0", dtype=torch.bfloat16], args=TensorList[Tensor[size=(5, 5, 5), device="cuda:0", dtype=torch.bfloat16], Tensor[size=(5, 5, 10), device="cuda:0", dtype=torch.bfloat16]], kwargs={'beta': '1', 'alpha': '1'}, broadcasts_input=False, name='')
```